cli/src/config/park-ui-config.ts

```diff
@@ -32,13 +32,15 @@ export const getConfig = (): Config => {
   try {
     const config = readFileSync(configPath, 'utf8')
     cachedConfig = JSON.parse(config)
-    if (!cachedConfig) {
-      throw new Error('Could not parse config file.')
-    }
-    return cachedConfig
   } catch (e) {
     throw new Error(`Could not find config file at ${configPath}.\nPlease run init first.`)
   }
+
+  if (!cachedConfig) {
+    throw new Error('Could not parse config file.')
+  }
```

Comment:
Fix: Line 36 does catch this error if we move where it was. Error message could be confusing and not true then.